merge-all.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+
+# Usage:
+# "REPOS_PATH=infrastructure GH_ACCESS_TOKEN=abc123 ./merge-all.rb"
+
+BRANCH_NAME = 'update-dependencies'
+
+def ruby_repos_file
+  File.open("#{ENV['REPOS_PATH']}/ruby")
+end
+
+def repos
+  [ruby_repos_file].map(&:to_a).flatten.map(&:strip).sort
+end
+
+# @return [Array<Hash>] the update PR
+def find_prs(client, repos)
+  repo_content = repos.map do |repo|
+    prs = client.pull_requests(repo, head: "sul-dlss:#{BRANCH_NAME}")
+
+    next if prs.empty?
```

Comment:
Should it bark if this is empty?

Reply:
No, that just means someone already merged that PR, so there's nothing to automatically do.

Reply:
It might also mean the PR wasn't created, which would indicate an error in jenkins (wrong bundler, etc.)?